README.md

```diff
@@ -1,8 +1,39 @@
 # NEURON Toolbox
-Interface for connecting NEURON and MATLAB.
 
-This main branch is intended to connect the two tools at a low level (C/C++) and avoid any third-party dependencies (e.g. work here will not require Python).
+Interface for connecting NEURON and MATLAB (on Windows, for MATLAB R2022a+).
 
-A separate branch, [using-python](https://github.com/mcdougallab/matlabneuroninterface/tree/using-python) at this point is more complete (in that it has an interface that can be used to generate an action potential) but it does the connection via Python.
+This exploration branch is intended to explore the possibilities for 
+connecting the two tools at a low level (C/C++) and avoid any third-party 
+dependencies (e.g. work here will not require Python).
+
+MATLAB is a registered trademark of The MathWorks, Inc. 
+
+## Usage
+
+First, make sure NEURON for Windows is installed (see http://neuron.yale.edu/).
+
+Run the MATLAB scripts in the following order:
+- **setup** 
+    - to add the appropriate directories to your path (you might need to
+      change the NEURON installation directory)
+    - to generate the library definition file
```

Comment:
from questions when emailing with mathworks: mention explicitly here that do not need to adapt generated definition file, even though some of it is outcommented

Reply:
Resolved in latest commit.